demo/tumor_concept/src/tumor_concept.h

```diff
@@ -72,7 +72,7 @@ struct GrowthModule : public BaseBiologyModule {
   void Run(T* cell) {
     if (cell->GetDiameter() < 8) {
       auto* random = TSimulation::GetActive()->GetRandom();
-      cell->ChangeVolume(400);
+      cell->ChangeVolume(400); // Here 400 is the speed and the change to the volume is based on the simulation time step, which by default is 0.01Hrs not 1s.
```

Comment:
The comment is exceeding our max line length.
I would recommend to put it on the line above and then run `make format`